common/pointers.go

```diff
@@ -13,3 +13,8 @@ func SafeStr(s *string) string {
 func SPtr(s string) *string {
 	return &s
 }
+
+// BPtr returns a pointer to the bool value b
+func BPtr(b bool) *bool {
```

Comment:
This functionality already exists in `bolt/pointers/pointers.go`. Not sure why we have two files that do the same thing...

Reply:
Nvm - this is sleet lol